cli/util/file_watcher.rs

```diff
@@ -163,6 +163,9 @@ pub struct WatcherCommunicator {
 
 impl WatcherCommunicator {
   pub fn watch_paths(&self, paths: Vec<PathBuf>) -> Result<(), AnyError> {
+    if paths.is_empty() {
+      return Ok(());
+    }
```

Comment:
What prompted this change?

Reply:
I moved it in here instead of being done only sometimes conditionally in the higher level code.